sql-odbc/src/IntegrationTests/ITODBCResults/test_odbc_results.cpp

```diff
@@ -936,7 +937,9 @@ TEST_F(TestSQLDescribeCol, SingleColumnMetadata) {
     EXPECT_EQ(single_col, m_column_name);
     EXPECT_EQ(single_col_name_length, m_column_name_length);
     EXPECT_EQ(single_col_data_type, m_data_type);
-    EXPECT_EQ(single_col_column_size, m_column_size);
+    // Value changes according to pagination setup on server
```

Comment:
Why is this?

Reply:
Not sure why. Created an issue #628 to investigate this further since I got one another value as 23 for server `1.9.0`. 

Will revert changes in this PR and handle this separately as it's not a blocker for fixing the data loading issue in Power BI.